{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by a third party damaging our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40782760",
  "multipleFaultNumbers": null,
  "date": "2020-10-23T19:53:30",
  "region": "Poulton-Le-Fylde",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-10-23T19:53:30",
  "information": "",
  "actualTimeOfRestoration": "2020-10-23T19:53:30",
  "outageCentrePoint": {
    "lat": 53.843858,
    "lng": -2.946057
  },
  "outageLocations": null,
  "AffectedPostcodes": " FY6 8LG, FY6 8LH, FY6 8LL, FY6 8LN, FY6 8LP, FY6 8LS, FY6 8LT, FY6 8LW, FY6 8NL, FY6 8NQ",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "D/CA"
}